### PR TITLE
Add 2.16 release notes

### PR DESCRIPTION
**Pull request by markusdregi**
Add release notes for 2.16

### CODE REVIEW

File: docs/rst/manual/about/release_notes.rst [hunk 1]
```diff
@@ -1,6 +1,54 @@
...
+
+For Equinor users we have installed a `run_external` command in the RMS
+environment that allows our users to reconstruct the environment prior to
+launching RMS to allow for executing commands within the standard user
+environment from RMS.
+
+2.16.0 ERT
+~~~~~~~~~~ 
+New features:
+  - Make it possible to run the IES2 via the command line interface
```
Comment (sondreso) [inline docs/rst/manual/about/release_notes.rst:26]:
```suggestion
  - Make it possible to run the IES via the command line interface
```
?

**Review by sondreso — APPROVED** [review]
LGTM!

**Comment by markusdregi** [comment]
Thanks @sondreso! Awaiting feedback from @oyvindeide as well and then I'll rebase the commits before merging.

**Comment by oyvindeide** [comment]
In regards to `semeio` there are a couple of more bugfixes, `design2params´ have a few parameters on deny list: 
```
DENYLIST = ["ENSEMBLE", "DATE"]
DENYLIST_DEFAULTS = DENYLIST + ["REAL"]
```
and the `stea` forward model is actually working now (it did not have a valid `ert` config before, and the name overlapped with `spinningjenny`)

`semeio.__version__` is also added and there have been documentation improvements. Also for `overburden_timeshift`(since 6.0.0): 

* Make horizon export optional
* Make ascii export optional

**Comment by markusdregi** [comment]
Thanks for the feedback @oyvindeide! I think I've added it now :)

Is it necessary to add the `__version__` to the release notes? Sounds very minor?

**Review by oyvindeide — APPROVED** [review]

**Pull request merged**